daffodil-propgen/src/main/resources/org/apache/daffodil/xsd/DFDL_part2_attributes.xsd

```diff
@@ -677,6 +677,8 @@
       type="dfdl:UTF16WidthEnum" />
     <xsd:attribute form="qualified" name="ignoreCase"
       type="dfdl:YesNoEnum" />
+    <xsd:attribute form="qualified" name="textBidi"
+      type="dfdl:YesNoEnum" />
```

Comment:
I looked for the other textBidi* properties and found they are defined further down in their own attributeGroup "TextBidiSchemeAGQualified" between lines 781-792.  If you moved this textBidi attribute further down the file to line 782 inside that group, would the test still work?  If so, please move it there to keep the textBidi* properties grouped together.